src/tools/tidy/src/deps.rs

```diff
@@ -54,10 +54,14 @@ pub(crate) const WORKSPACES: &[(&str, ExceptionList, Option<(&[&str], &[&str])>)
         Some((&["rustc_codegen_cranelift"], PERMITTED_CRANELIFT_DEPENDENCIES)),
     ),
     // tidy-alphabetical-start
-    //("compiler/rustc_codegen_gcc", EXCEPTIONS_GCC, None), // FIXME uncomment once all deps are vendored
-    //("library/backtrace", &[], None), // FIXME uncomment once rust-lang/backtrace#562 has been synced back to the rust repo
-    //("library/portable-simd", &[], None), // FIXME uncomment once rust-lang/portable-simd#363 has been synced back to the rust repo
-    //("library/stdarch", EXCEPTIONS_STDARCH, None), // FIXME uncomment once rust-lang/stdarch#1462 has been synced back to the rust repo
+    //("compiler/rustc_codegen_gcc", EXCEPTIONS_GCC, None), // FIXME uncomment once
+    //                                                         all deps are vendored
+    //("library/backtrace", &[], None), // FIXME uncomment once rust-lang/backtrace#562
+    //                                     has been synced back to the rust repo
+    //("library/portable-simd", &[], None), // FIXME uncomment once rust-lang/portable-simd#363
+    //                                         has been synced back to the rust repo
+    //("library/stdarch", EXCEPTIONS_STDARCH, None), // FIXME uncomment once rust-lang/stdarch#1462
+    //                                                  has been synced back to the rust repo
```

Comment:
So, I wanted to look at what logic tidy applies and I see this... that doesn't bode well for the rest of the PR

Reply:
> So, I wanted to look at what logic tidy applies and I see this...

Tidy only checks, doesn't modify/bless.

That was done manually. How would you prefer to shorten that comment? There aren't many alternatives.

Reply:
I assume the line-length rule is supposed to improve readability. If it fails to achieve the goal then the block should be exempted from that.